app/main/views.py

```diff
@@ -87,25 +85,26 @@ def update_service(service_id):
     db.session.add(service)
     db.session.commit()
 
-    return jsonify(services=jsonify_service(service)), 200
+    return jsonify(services=jsonify_service(service)), http_status
```

Comment:
A possible alternative to returning the whole JSON in the response might be to instead set the "Content-Location" header to the URL where the created/updated resource can be accessed.   e.g. a PUT to `/services/12345` would return 
`200 OK
Content-Location: /services/12345`